src/main/java/trade/wayruha/mexc/service/endpoint/MarketDataAPI.java

```diff
@@ -1,11 +1,25 @@
 package trade.wayruha.mexc.service.endpoint;
 
+import org.jetbrains.annotations.NotNull;
 import retrofit2.Call;
 import retrofit2.http.GET;
+import retrofit2.http.Query;
 import trade.wayruha.mexc.dto.ExchangeInfo;
+import trade.wayruha.mexc.dto.TickerPrice;
+
+import java.util.List;
+
+import static trade.wayruha.mexc.constant.ApiEndpoints.SYMBOL_PRICE_TICKER_API_PATH;
 
 public interface MarketDataAPI {
 
     @GET("api/v3/exchangeInfo")
     Call<ExchangeInfo> getExchangeInfo();
+
+    /** If the symbol is not sent, all symbols will be returned in an array.*/
+    @GET(SYMBOL_PRICE_TICKER_API_PATH)
```

Comment:
Actually, I'd prefer having URL to be stored here, since it makes less actions while navigating or location necessary URL as well as help to see version mismatch sometimes.
It's generally more efficient (IMHO) to have URLs placed next to the descriptive method names.

Reply:
As you prefer